components/UploadForm.vue

```diff
@@ -347,12 +360,25 @@ const processEPub = async ({ buffer, file }: { buffer: ArrayBuffer; file: File }
     }
     epubMetadataList.value.push(epubMetadata)
   } catch (err) {
+    // eslint-disable-next-line no-console
     console.error(err)
   }
 }
 
 const handleDeleteFile = (index: number) => {
-  fileRecords.value.splice(index, 1)
+  const removedFile = fileRecords.value.splice(index, 1)[0]
```

Comment:
```suggestion
  const [removedFile] = fileRecords.value.splice(index, 1)
```
Then maybe early return if `removedFile` is undefined or invalid

Reply:
[♻️ Simplify assignment using array destructuring](https://github.com/likecoin/nft-book-press/pull/354/commits/61e5804db64df2a2c2d5745608f05ec233555b7d)